android-sdk/tasks/main.yml

```diff
@@ -77,8 +77,8 @@
 
 - debug: var=accept
 
-- pause:
-    prompt: "Licences are up to date. Hit enter to continue with the playbook "
+- debug:
+    msg: "Licences are up to date. Continuing...."
   when: accept is undefined
 
```

Comment:
This is fine but lets remove the set_fact taks above this and change the when to when: "'Accept?' not in rsh_cmd.stdout"

We have an additional task here for no reason